### ISSUE TITLE
EDT не дает удалить документ, если он входит в состав типа параметра общей команды

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#113]
### Описание ошибки

Документ входит в состав типа параметра общей команды. Пытаюсь удалить документ, но кнопка Готово недоступна. Отматываю диалог назад, вижу сообщение о "конфликтах". Конфигуратор с такой задачей легко справляется.
В данном примере тип параметра команды составной и от удаления одного из документов ничего по сути не меняется. Почему Edt этого не может сделать - не понятно.

### Как воспроизвести

[test.zip](https://github.com/1C-Company/1c-edt-issues/files/7565939/test.zip)
[EDT-test.zip](https://github.com/1C-Company/1c-edt-issues/files/7565943/EDT-test.zip)

1. Можно использовать выгрузку информационной базы test.zip или проект EDT-test.zip
2. Попробуйте удалить в EDT Документ1 или Документ2
3. При составном типе параметр команды не останется пустым. Почему Edt не может удалить один из документов не понятно.

### Скриншоты

![image](https://user-images.githubusercontent.com/54451084/142496361-2986ea0b-9efa-4784-a8c7-35425d13e65a.png)

![image](https://user-images.githubusercontent.com/54451084/142496394-d8861be5-ef21-425a-9e03-f84ae87e332b.png)

### Ожидаемое поведение

Если документ не является последним в параметре команды - давать удалять его и очищать ссылку из параметра команды как это делает конфигуратор.

### Лог рабочей области

[логи.zip](https://github.com/1C-Company/1c-edt-issues/files/7565954/default.zip)

### Версия 1С:EDT

Ruby 2021.2.10

### Операционная система

Windows

### Установленные плагины

1C:Code style V8, 1C:SSL-support

**Comment by DmitryShvaika** [comment]
Извиняюсь - конфигуратор так же не дает удалить в этом случае документ - задача закрыта

**Comment by marmyshev** [comment]
Имхо, неплохое пожелание.

Если тип в общей команде составной - то можно было бы чистить состав типа команды. А если тип единственный - предлагать удалить заодно и общую команду.

**Comment by DmitryShvaika** [comment]
> А если тип единственный - предлагать удалить заодно и общую команду.

Так не надо... лучше пусть не удаляет как сейчас. "Заодно" ничего делать не надо. Пользователь хочет удалить именно документ. Значит удаляться должен только документ. Иначе чехарда получится.

**Comment by DmitryShvaika** [comment]
Думаю, даже плохая идея "заодно" менять составной тип команды. Наверное то как работает сейчас - самый правильный вариант. Или в диалоге рефакторинга надо выводить, что будет изменен тип команды с такого-то на такой-то. Ну или сообщением, что документ будет удален из команды. И давать убрать или поставить галочку... если галка убрана - не давать удалять.

**Comment by marmyshev** [comment]
> Или в диалоге рефакторинга надо выводить, что будет изменен тип команды с такого-то на такой-то

Да, я имел ввиду "предлагать" пользователю - это как раз значит отдельный пункт в дереве Delete-рефакторинга - по поводу чистки типов. 

Предполагаю, что его даже необходимо по умолчанию предлагать с выключенной галкой.
